Misc/NEWS.d/next/Core and Builtins/2018-02-27-20-57-00.bpo-32911.cmKfco.rst

```diff
@@ -0,0 +1,4 @@
+Reverted :issue:`29463`. ``docstring`` field is removed from Module, ClassDef,
```

Comment:
We should say why we are doing this.  Perhaps start the entry with:
"Due to unexpected compatibility issues discovered during downstream beta testing, reverted ... "